component/pom.xml

```diff
@@ -141,6 +141,11 @@
             <groupId>org.apache.kafka</groupId>
             <artifactId>connect-api</artifactId>
         </dependency>
+
```

Comment:
remove unnecessary new line

Reply:
Fixed in 9b553da8e0ca2f9d10e0868734562af8cfac30e7